### DISCUSSION TITLE
Install in existing theme with css.liquid files

### DISCUSSION THREAD

**Discussion by anders-christiansen** [barrel/shopify-vite#86] [Q&A] [answered]
I'm trying to use shopify-vite for an existing theme which contains a css.liquid file. I've moved the js and css files to frontend/entrypoints, but the css.liquid file giving me trouble. Do i need to refactor so that no liquid is used in the css files, or is there a work around for this? :)

**Comment by montalvomiguelo** [comment, marked as answer]
@anders-christiansen Yeah prolly you will need to do some refactoring. Typically the liquid bits in that file output settings like colors, fonts, etc. You could [extract those pieces](https://github.com/montalvomiguelo/hydrogen-theme/blob/main/layout/theme.liquid#L40-L59) to your layout file or a separate snippet.
